pekko/persistence/src/test/scala/org/apache/pekko/persistence/kafka/journal/PersistentBinaryToBytesSpec.scala

```diff
@@ -0,0 +1,25 @@
+package org.apache.pekko.persistence.kafka.journal
+
+import cats.syntax.all.*
+import com.evolution.serialization.SerializedMsg
+import com.evolutiongaming.kafka.journal.SerdeTesting
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+class PersistentBinaryToBytesSpec extends AnyFunSuite with Matchers with SerdeTesting {
+
+  test("toBytes & fromBytes") {
+
+    val expected = PersistentBinary(
+      manifest = "persistentManifest".some,
+      writerUuid = "writerUuid",
+      payload = SerializedMsg(identifier = 2, manifest = "manifest", bytes = "payload".encodeUtf8Unsafe),
+    )
+
+    verifyEncodeDecodeExample(
+      valueExample = expected,
+      encodedExampleFileName = "PersistentBinary.bin",
+//      dumpEncoded = true,
```

Comment:
Would be good to remove this line

Reply:
this line is in original code - it is used to generate initial binary file for new unit-tests, IIRC